### PR TITLE
Update user guide to ensure consistency

### LINKED ISSUE #347: Update user guide for overall consistency

### PR DESCRIPTION
**Pull request by ngjianyi**
Closes #347

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -448,38 +471,38 @@ Expected output:
...
 
-<box type="warning" seamless>
+<box type="caution" seamless>
 
 **Warnings**:
 - At least one space is needed between `mark` and `-d`.
 - At least one space is needed between `-d` and `INDEX`.
-- At least one space is needed between `INDEX` and `STATUS.
+- At least one space is needed between `INDEX` and `STATUS`.
+- All parameters must be given.
```
Comment (prave1n) [inline docs/UserGuide.md:492]: there is only one parameter for this command which is `INDEX` so this may not be needed

File: docs/UserGuide.md [hunk 2]
```diff
@@ -448,38 +471,38 @@ Expected output:
...
+- At least one space is needed between parameters.
```
Comment (prave1n) [inline docs/UserGuide.md:493]: there is only one parameter for this command which is `INDEX` so this may not be needed

File: docs/UserGuide.md [hunk 3]
```diff
@@ -529,21 +560,28 @@ Expected output:
...
 Format: `sort -d so/SORT_ORDER sb/SORT_BY`
 
-<box type="details" seamless>
+<box type="info" seamless>
 
 Parameters:
 
-- `so/SORT_ORDER`: `SORT_ORDER` must be either `a` for ascending or `d` for descending, and must not be blank.
-- `sb/SORT_BY`: `SORT_BY` must be either `c` for cost, `d` for date and time or `s` for status, and must not be blank.
+- `SORT_ORDER`: The sorting order. It must be either `a` for ascending or `d` for descending, and cannot be blank.
```
Comment (prave1n) [inline docs/UserGuide.md:571]: should the parameter include so/ and sb/

File: docs/UserGuide.md [hunk 4]
```diff
@@ -529,21 +560,28 @@ Expected output:
...
+    - `SORT_ORDER` is case-sensitive.
```
Comment (prave1n) [inline docs/UserGuide.md:572]: since this is mentioned in the warnings, this may seem repetitive

**Review by LinkesV — APPROVED** [review]
LGTM

**Review by prave1n — COMMENTED** [review]
Overall, looks more standardised. Some minor changes may be required

File: docs/UserGuide.md [hunk 5]
```diff
@@ -105,19 +106,21 @@ Format: `help`
...
 **Warnings**:
 - No other parameters should be given for this command.
 - Give at least one space between list and -a.
+
 </box>
 
 ### Clearing all suppliers and deliveries : `clear`
 
-Clear all data regarding suppliers and deliveries in vendor vault.
+Clears all data regarding suppliers and deliveries in vendor vault.
```
Comment (CJianzhi) [inline docs/UserGuide.md:123]: VendorVault should be used here

File: docs/UserGuide.md [hunk 6]
```diff
@@ -144,20 +146,26 @@ Parameters:
...
+  - A `PRODUCT` cannot be made up of only spaces.
+
+</box>
+
+<box type="tip" seamless>
 
-**Tip:** A supplier can have any number of `TAG` and `PRODUCT` (including 0).
-- To include multiple `TAG`, use multiple `t/TAG`.
+**Tips:** A supplier can have any number of `TAG` and `PRODUCT` (including 0).
+- To include multiple `TAG`/`PRODUCT`, use multiple `t/TAG`/`pro/PRODUCT` respectively.
```
[the remark this answers is not in the document]
Comment (CJianzhi) [reply docs/UserGuide.md:156]: I think it will look funny with the s behind tho in markdown

File: docs/UserGuide.md [hunk 7]
```diff
@@ -197,127 +205,139 @@ Deletes a supplier from the list of suppliers in VendorVault.
...
 Format: `mark -s INDEX STATUS`
 
-<box type="details" seamless>
+<box type="info" seamless>
 
 Parameters:
-- `INDEX`: The index of the supplier in the list. Must be a number greater than 0 and must not be blank.
-- `STATUS`: Must be one of the following: `active`, `inactive` and must not be blank.
+- `INDEX`: The index of the supplier to be marked in the displayed list. It must be a number between 1 and the total number of suppliers displayed (inclusive), and cannot be blank.
+- `STATUS`: The suppler status to change to. It must be one of the following values: `active`, `inactive`, and cannot be blank.
```
Comment (CJianzhi) [inline docs/UserGuide.md:245]: Typo here for supplier

File: docs/UserGuide.md [hunk 8]
```diff
@@ -197,127 +205,139 @@ Deletes a supplier from the list of suppliers in VendorVault.
...
+</box>
 
-    mark -s 3 active
+Examples:
+- `mark -s 3 inactive`
+- Sorting by name using `sort -s so/a sb/n` followed by `mark -s 1 inactive`
 
-A success message will be displayed if the supplier is successfully marked as active.
+Expected outputs:
+- Supplier at index 3 of the displayed list has status marked as `active`, assuming there is at least three suppliers displayed and the third supplier is not currently `INACTIVE`. Otherwise, an error message will be shown.
```
Comment (CJianzhi) [inline docs/UserGuide.md:270]: Should be not currently `ACTIVE`, and should the status marked as `active` be in caps as well

File: docs/UserGuide.md [hunk 9]
```diff
@@ -197,127 +205,139 @@ Deletes a supplier from the list of suppliers in VendorVault.
...
-#### Example
-To find the supplier whose name contains "link" and company contains "NU":
+</box>
 
-    find -s n/link com/NU
+Example:
+- `find -s n/link com/NU`
 
+Expected output:
+- All suppliers with `name` containing `link` and `COMPANY` containing `NU` is shown.
```
Comment (CJianzhi) [inline docs/UserGuide.md:320]: grammatical error, should use "are shown" i think

File: docs/UserGuide.md [hunk 10]
```diff
@@ -197,127 +205,139 @@ Deletes a supplier from the list of suppliers in VendorVault.
...
+Sorts suppliers based on their name in VendorVault.
 
 Format: `sort -s so/SORT_ORDER sb/SORT_BY`
 
+<box type="info" seamless>
+
 Parameters:
-- `SORT_ORDER`: Must be either 'a' for ascending or 'd' for descending.
-- `SORT_BY`: Must be 'n' for name. (Current version of VendorVault only supports sorting by name)
+- `SORT_ORDER`: The sorting order. It must be either `a` for ascending or `d` for descending, and cannot be blank.
```
Comment (CJianzhi) [inline docs/UserGuide.md:334]: need add prefix in front for each parameter

File: docs/UserGuide.md [hunk 11]
```diff
@@ -366,19 +384,23 @@ Parameters:
...
 **Tips:** 
 - Day, month, hour and minute of DELIVERY_DATE_TIME must be in double digits!
   - example: 03-03-2024 02:01
 
 </box>
 
-<box type="warning" seamless>
+<box type="caution" seamless>
 
 **Warnings**:
```
Comment (CJianzhi) [inline docs/UserGuide.md:399]: Should we mention no duplicate parameters for all commands?

File: docs/UserGuide.md [hunk 12]
```diff
@@ -529,21 +560,28 @@ Expected output:
...
-<box type="details" seamless>
+<box type="info" seamless>
 
 Parameters:
 
-- `so/SORT_ORDER`: `SORT_ORDER` must be either `a` for ascending or `d` for descending, and must not be blank.
-- `sb/SORT_BY`: `SORT_BY` must be either `c` for cost, `d` for date and time or `s` for status, and must not be blank.
+- `SORT_ORDER`: The sorting order. It must be either `a` for ascending or `d` for descending, and cannot be blank.
+    - `SORT_ORDER` is case-sensitive.
+- `SORT_BY`: The field to sort by. It must be either `c` for `COST`, `d` for `DELIVERY_DATE_TIME` or `s` for `STATUS`, and cannot be blank.
```
Comment (CJianzhi) [inline docs/UserGuide.md:573]: Parameter does not have prefix

**Review by CJianzhi — COMMENTED** [review]
LGTM! Just some minor errors that needs to be edited

File: docs/UserGuide.md [hunk 13]
```diff
@@ -343,8 +347,8 @@ Examples:
 - Sorting by name using `sort -s so/a sb/n` followed by `mark -s 1 inactive`
 
 Expected outputs:
-- Supplier at index 3 of the displayed list has status marked as `active`, assuming there is at least three suppliers displayed and the third supplier is not currently `INACTIVE`. Otherwise, an error message will be shown.
-- Supplier at index 1 of the displayed list has status marked as `inactive`, assuming there is at least one supplier displayed and the first supplier is not currently `INACTIVE`. Otherwise, an error message wil be shown.
+- Supplier at index 3 of the displayed list has status marked as `active`, assuming there is at least three suppliers displayed and the third supplier is not currently `inactive`. Otherwise, an error message will be shown.
```
Comment (CJianzhi) [inline docs/UserGuide.md:350]: I thought it should be the supplier that is active cannot be marked as  active again

**Review by CJianzhi — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 14]
```diff
@@ -413,19 +433,17 @@ Parameters:
 - All parameters must be given.
 - No duplicate parameters can be used.
 - `SORT_ORDER` and `SORT_BY` are case-sensitive.
-- The sort command will only sort all suppliers in VendorVault.
-- i.e.
-    - If you have searched for a supplier using the `find` command,
-      the sort command will not sort the previous results of the `find` command, but will sort all suppliers in VendorVault.
-</box>
+- The sort command will sort all suppliers in VendorVault, not just the displayed list.
+  - e.g. If you have searched for suppliers using the `find` command, the sort command will not sort the previous results of the `find` command, but will sort all suppliers in VendorVault.
```
Comment (CJianzhi) [inline docs/UserGuide.md:437]: Should this be `find -s` command?

File: docs/UserGuide.md [hunk 15]
```diff
@@ -448,38 +471,38 @@ Expected output:
...
+- `STATUS`: The delivery status to change to. It must be one of the following values: `PENDING`, `DELIVERED`, `CANCELLED`, and cannot be blank.
 
 </box>
 
-<box type="warning" seamless>
+<box type="caution" seamless>
 
 **Warnings**:
 - At least one space is needed between `mark` and `-d`.
 - At least one space is needed between `-d` and `INDEX`.
```
[the remark this answers is not in the document]
Comment (CJianzhi) [reply docs/UserGuide.md:591]: Need to be updated for this point

File: docs/UserGuide.md [hunk 16]
```diff
@@ -638,21 +669,27 @@ Expected output:
 
 ### 3.4.6 Sorting deliveries: `sort -d`
 
-The `sort -d` command is used to sort deliveries in VendorVault based on the delivery cost, date and time or status.
-This helps you to view the deliveries in a different order (ascending or descending).
+Sorts deliveries based on their cost, date and time, and status in VendorVault.
```
Comment (CJianzhi) [inline docs/UserGuide.md:672]: Should this be or instead of and?

**Review by CJianzhi — CHANGES_REQUESTED** [review]
LGTM! Some minor changes needed before merging

**Review by CJianzhi — APPROVED** [review]
LGTM!

**Pull request merged**